### PR TITLE
Population annotation

### LINKED ISSUE #52: Add population annotation to match other safety explorer renderers.

### LINKED ISSUE #92: Indicate # of hidden inliers for each panel
# Test notes
* When the _Inliers_ checkbox is unchecked a box in the lower left of each chart displays the number of inliers for that measure.

### PR DESCRIPTION
**Pull request by samussiah**
# Issues
Closes #52 
Closes #92

### CODE REVIEW

File: build/paneledOutlierExplorer.js [hunk 1]
```diff
@@ -621,11 +622,26 @@
...
             }
         });
+
+        //If data do not have normal range variables update normal range method setting and options.
+        if (
+            this.config.variables.indexOf(this.config.lln_col) < 0 ||
+            this.config.variables.indexOf(this.config.uln_col) < 0
+        ) {
+            if (this.config.normal_range_method === 'LLN-ULN')
+                this.config.normal_range_method = 'Standard Deviation';
```
Comment (pburnsdata) [inline build/paneledOutlierExplorer.js]: not sure if this was suppose to have brackets after it

File: src/charts/onResize/brush/brushMarks.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    this.parent.wrap
+        .selectAll('.line-supergroup g.line path')
+        .classed('brushed', false)
+        .filter(d => this.parent.data.selectedIDs.indexOf(d.id) > -1)
+        .classed('brushed', true)
+        .each(function(d) {
+            select(this.parentNode).moveToFront();
+        });
+
+    //Draw listing displaying brushed IDs first.
```
Comment (pburnsdata) [inline src/charts/onResize/brush/brushMarks.js:47]: Does this mean listing should have all the ids, with the brushed ones first, or that it should only have the brushed ones? Right now I think it only has the brushed ids.

Comment (samussiah) [reply src/charts/onResize/brush/brushMarks.js:47]: Just the brushed.  It was a little bit of a pain to update the sort of the listing without updating the data.

**Comment by brittsikora** [comment]
Pass - the inliers count is appearing as expected and the population annotation (x % of x show) is working as well

**Review by brittsikora — APPROVED** [review]

**Pull request merged**